### DISCUSSION TITLE
Appropriate model size

### DISCUSSION THREAD

**Discussion by 34j** [voicepaw/so-vits-svc-fork#314] [Q&A]
I know the model parameters have not changed much from the original VITS (for 100 speakers), but is it appropriate to keep the model size to train a model for one speaker?

**Comment by Lordmau5** [comment]
I did actually want to ask about the model parameters as I'm personally completely unsure what could be changed to alter training in both a good and bad way.

`learning_rate` for example. Is that good to have on the default 0.0001 or would 0.0005 be better, perhaps even 0.05?

As for all the other stuff further down in the `"model"` section, such as `inter_channels`, `hidden_channels`, etc., I have absolutely no idea.

I couldn't find any documentation about those parameters :/

**Reply by 34j** [reply]
I think learning_rate is currenly too high for a small dataset, but it might also be related to batch size
